### PR TITLE
Add initial helm chart

### PR DESCRIPTION
**Pull request by MarcelMue**
This PR adds all the helm foo, idea here is to get a helm chart which I want to test on lycan with `deployer`.

### CODE REVIEW

**Comment by MarcelMue** [comment]
This is very close to the helm setup in `kvm-operator` as I am not aware if I need to do anything special.

**Comment by MarcelMue** [comment]
Tested this on `lycan`, pod came up fine and started doing its thing. I scaled it down for the time and will continue working on rolling out `endpoint-operator` tomorrow.

File: helm/endpoint-operator-chart/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+            path: /healthz
+            port: 8000
+          initialDelaySeconds: 30
+          timeoutSeconds: 1
+        resources:
+          requests:
+            cpu: 100m
+            memory: 20Mi
+          limits:
+            cpu: 250m
```
Comment (JosephSalisbury) [inline helm/endpoint-operator-chart/templates/deployment.yaml]: Let's make requests and limits the same, so we get the guaranteed scheduling goodness

Comment (MarcelMue) [reply helm/endpoint-operator-chart/templates/deployment.yaml]: I added newlines and set the same `request` and `limit` resources, unsure if they are reasonable.

**Pull request merged**